helm/aws-operator-chart/templates/deployment.yaml

```diff
@@ -1,50 +1,53 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: aws-operator
-  namespace: {{ .Values.namespace }}
+  name: {{ tpl .Values.resource.default.name  . }}
+  namespace: {{ tpl .Values.resource.default.namespace  . }}
   labels:
-    app: aws-operator
+    app: {{ tpl .Values.resource.default.name  . }}
 spec:
   replicas: 1
   revisionHistoryLimit: 3
+  selector:
+    matchLabels:
+      app: {{ tpl .Values.resource.default.name  . }}
   strategy:
     type: RollingUpdate
   template:
     metadata:
       annotations:
         releasetime: {{ $.Release.Time }}
       labels:
-        app: aws-operator
+        app: {{ tpl .Values.resource.default.name  . }}
     spec:
       volumes:
-      - name: aws-operator-configmap
+      - name: {{ .Values.project.name }}-configmap
         configMap:
-          name: aws-operator-configmap
+          name: {{ tpl .Values.resource.default.name  . }}
           items:
           - key: config.yaml
             path: config.yaml
-      - name: aws-operator-secret
+      - name: {{ .Values.project.name }}-secret
         secret:
-          secretName: aws-operator-secret
+          secretName: {{ tpl .Values.resource.default.name  . }}
           items:
           - key: secret.yaml
             path: secret.yaml
       - name: certs
         hostPath:
           path: /etc/ssl/certs/ca-certificates.crt
-      serviceAccountName: aws-operator
+      serviceAccountName: {{ tpl .Values.resource.default.name  . }}
       securityContext:
-        runAsUser: {{ .Values.userID }}
-        runAsGroup: {{ .Values.groupID }}
+        runAsUser: {{ .Values.pod.user.id }}
+        runAsGroup: {{ .Values.pod.group.id }}
       containers:
-      - name: aws-operator
-        image: quay.io/giantswarm/aws-operator:[[ .SHA ]]
+      - name: {{ .Values.project.name }}
+        image: "{{ .Values.image.registry }}/{{ .Values.image.name }}:{{ .Values.image.tag }}"
```

Comment:
Should `.Values.image.registry` be `.Values.Installation.V1.Registry.Domain` ?

Reply:
Good call. I'll fix this now.

Reply:
This should be fixed now.